variables.tf

```diff
@@ -630,3 +630,17 @@ variable "drop_invalid_header_fields" {
 
   default = false
 }
+
+variable "cloudwatch_agent_system_config" {
+  description = "Cloudwatch Agent Config for system metrics"
+  type        = string
+
+  default = ""
```

Comment:
let's either do `default = "no-set"` or make role statements above conditional

Reply:
I'll set a non-existent keyname here as TF to do a conditional policy block is erroring when doing a conditional on `var.cloudwatch_agent_system_config != ""`